### ISSUE TITLE
memory access error on server.py

### ISSUE THREAD

**Issue by Flomedi** [Freenove/Freenove_4WD_Smart_Car_Kit_for_Raspberry_Pi#39]
I have the problem that i get a "memory access error" (Speicherzugriffsfehler in german) whenever i try to connect to the server. In windows i get "unpack requires a buffer of 4 bytes" error. I provide screenshots for more clearity.
![cmd](https://user-images.githubusercontent.com/82578574/209138221-588e7603-4d1a-45c1-ae04-bc21aec09116.png)
![raspberry](https://user-images.githubusercontent.com/82578574/209138223-80a01e0d-db6b-497b-a2ed-1d5e1b2c6bdd.png)

**Comment by Shaynee-MO** [comment]
Please install the latest 32-bit Raspberry Pi OS and try again.
https://www.raspberrypi.com/software/operating-systems/

![1](https://user-images.githubusercontent.com/118493923/209631063-fea7d3a5-a35a-4d5a-aac8-3545bb9befc8.png)

**Comment by Blcker12** [comment]
I have the same issue and it came back after installing the latest greatest image and all updates.
![grafik](https://user-images.githubusercontent.com/122253793/211250303-5afda7be-8c06-493b-af70-39f7fa95a8ae.png)

**Comment by Blcker12** [comment]
it is an issue with the picamera2 configuration. When I put the camera part into comments (server.py) the issues is gone. 
Need to check but I think the output configuration is causing it.

**Comment by fluxens** [comment]
Did you solve this problem?

**Comment by Blcker12** [comment]
Yes. First I got an Raspy Image from the developers which was working. After that I re-installed the orifinal image from www.raspberrypi.com/software/ with the Raspberry imager programm and it work as well... But I still have no clue what was wrong at first trial...

**Comment by fluxens** [comment]
@fluxens thanks for the reply. I updated the image and it works for me now as well.

I think this issue can be closed.
